### PR TITLE
Refactor Compose Multiplatform benchmarks to remove code duplication.

### PR DESCRIPTION
**Pull request by pjBooms**
- kn+performance and jvm-vs-kotlin-native benchmarks are merged into one multiplatform where they measure both CPU time and missed frames + frames percentiles

- added multiplatform run arguments processing for benchmarks configuration

- deleted standalone samples (moved LazyList to the infrastructure)

Fixes [CMP-5790](https://youtrack.jetbrains.com/issue/CMP-5790/Refactor-CMP-benchmarks-to-remove-code-duplication)

### CODE REVIEW

File: benchmarks/multiplatform/build.gradle.kts [hunk 1]
```diff
@@ -21,26 +23,21 @@ kotlin {
...
-                    "-linker-option", "-framework", "-linker-option", "Metal"
-                )
             }
         }
     }
 
     wasmJs {
-        browser()
         binaries.executable()
+        browser {
```
Comment (elijah-semyonov) [inline benchmarks/multiplatform/build.gradle.kts:39]: Was something intended to be added there?

**Review by elijah-semyonov — APPROVED** [review]
LGTM

**Review by eymar — APPROVED** [review]
Thank you!
Can we swtich to kotlin 2.1.0? So wasmJs target would not fail to compile?

File: benchmarks/multiplatform/build.gradle.kts (hunk 1 above)
Comment (pjBooms) [reply benchmarks/multiplatform/build.gradle.kts]: Yes, it was. Fixed  :)

**Pull request merged**